src/context_flags.ml

```diff
@@ -0,0 +1,22 @@
+#directory "+compiler-libs";;
+#load "ocamlcommon.cma";;
+
+match Sys.argv.(1) with
+| "cxxflags" ->
+    if Sys.win32 && Config.ccomp_type = "msvc" then
+      print_string "(/EHsc)"
```

Comment:
MSVC warned that this is required, so I added it - it's got something to do with exception unwinding!